vendors/cypress/bluetooth/psoc6/cyosal/src/cybt_osal_amzn_freertos.c

```diff
@@ -50,11 +50,9 @@ const uint32_t millis_to_tick_ratio = (uint32_t)( 1000 / configTICK_RATE_HZ );
  *               Function Definitions
  ******************************************************/
 
-extern signed portBASE_TYPE xTaskIsTaskFinished( TaskHandle_t xTask ) PRIVILEGED_FUNCTION;
-
 wiced_result_t wiced_rtos_init_semaphore( wiced_semaphore_t* sem )
 {
-    SemaphoreHandle_t xsemaphore = NULL;
+    SemaphoreHandle_t* xsemaphore = NULL;
```

Comment:
Why was this changed?  `xSemaphoreCreateCounting()` (line 61) [returns `SemaphoreHandle_t`](https://www.freertos.org/CreateCounting.html), not `SemaphoreHandle_t *`.  It looks like this line should be changed back to the original version.

Reply:
You are correct. Will Fix. Thanks.